EfEnumToLookup/LookupGenerator/SqlServerHandler.cs

```diff
@@ -100,7 +100,7 @@ private string AddForeignKeys(IEnumerable<EnumReference> refs)
 				var fkName = string.Format("FK_{0}_{1}", enumReference.ReferencingTable, enumReference.ReferencingField);
 
 				sql.AppendFormat(
-					" IF OBJECT_ID('{0}', 'F') IS NULL ALTER TABLE [{1}] ADD CONSTRAINT {0} FOREIGN KEY ([{2}]) REFERENCES [{3}] (Id);\r\n",
+					" IF OBJECT_ID('{0}', 'F') IS NULL AND OBJECT_ID('{1}', 'U') IS NOT NULL ALTER TABLE [{1}] ADD CONSTRAINT {0} FOREIGN KEY ([{2}]) REFERENCES [{3}] (Id);\r\n",
```

Comment:
Neat solution!

It bothers me a little that the model we have in memory still thinks there's a fk, which might make future work on the library more challenging. Having said that I guess EF is unaware it's a view so we're stuffed. I'll take this solution as I don't have any better ideas :-)